### DISCUSSION TITLE
Não está rendenizando as páginas >> TagHelpers <<

### DISCUSSION THREAD

**Discussion by PriciLeite** [balta-io/forum#292] [Tira Dúvidas] [answered]
Olá, Boa tarde!
Pderia ajudar a entender pq não está rendenizando as páginas? 
![duvida](https://user-images.githubusercontent.com/109990443/227741242-b2416eb6-6195-41ae-92bd-4da6f05c3a3f.png)

<partial name=" " -> não reconhece partial... 
Obrigada!

**Comment by PriciLeite** [comment]
![duvida](https://user-images.githubusercontent.com/109990443/227741390-709ec5ef-cf86-459e-b53e-2aad67209172.png)

**Comment by PriciLeite** [comment, marked as answer]
![duvida](https://user-images.githubusercontent.com/109990443/227741526-80233fa3-cae2-481b-a01b-98e80a2e288f.png)

**Reply by andrebaltieri** [reply]
Bom dia, @PriciLeite, funcionou?

**Comment by PriciLeite** [comment]
Bom dia Balta! 
Para esse  caso do <partial name="Shared/NavMenuPartial" /> parece ter identificando pq ele rendenizou...
No entanto ao criar _Layout e passando no index.cshtml e adicionando @RenderBody( ) em _Layout ele dá erro... 
 @{
    Layout = "Shared/_Layout";
}
Ele dá um erro diferente do seu... No seu foi necessário só adicionar apenas @RenderBody( ) no _Loyout... para corrigir o erro...
vou te mandar uma imagem...

Index.cshtml
![duvida](https://user-images.githubusercontent.com/109990443/227976271-7a7f471f-95b5-489d-b950-0f30f3291b10.png)


_Layout 
![duvida](https://user-images.githubusercontent.com/109990443/227977098-d9f8ebe5-4e71-48af-9b38-1d18f5bd8aac.png)


Erro
![duvida](https://user-images.githubusercontent.com/109990443/227978151-715c0c93-e125-4f0e-8c63-a2b895239856.png)

**Comment by PriciLeite** [comment]
Não consigo enteder como ele não está achando com as referências... :/

**Reply by andrebaltieri** [reply]
@PriciLeite boa tarde... no caso, acredito que o erro seja por conta do `@model Index`... por favor, tenta especificar o caminho completo do seu model aí... acredito que ele está se confundindo e pegando outro `index`.

**Reply by PriciLeite** [reply]
Corrigi para todas as páginas...

@model MyRazorApp.Pages.Index
@model MyRazorApp.Pages.Login
@model MyRazorApp.Pages.About

não deu certo :(

**Reply by PriciLeite** [reply]
Professor deu certo aqui 👍🏻 era o model do  _ViewImports 
Obrigada...

**Reply by andrebaltieri** [reply]
Que bom @PriciLeite 🚀
